run_tests.sh

```diff
@@ -1,6 +1,13 @@
 #!/bin/bash
 set -e
+cd ..
+DICTCOMMIT=`git rev-parse HEAD` && echo "DICTCOMMIT=\"${DICTCOMMIT}\"" >dictionaryutils/dictionaryutils/version_data.py
+DICTVERSION=`git describe --always --tags` && echo "DICTVERSION=\"${DICTVERSION}\"" >>dictionaryutils/dictionaryutils/version_data.py
+cd dictionaryutils
 pip install -r dev-requirements.txt
+# always use this version of dictionaryutils...
+pip uninstall -y dictionaryutils
+python setup.py install --force
```

Comment:
if you run `setup.py develop` instead of `setup.py install --force`, it will create an egg-link back to the source that's already pulled (saving space). this will also make it so you can modify the source and run tests again without the need to uninstall/reinstall (though this script will relink anyway, but it will be faster since it's not copying all the source files)

Reply:
See https://github.com/uc-cdis/datadictionary/blob/develop/.travis.yml#L6 for how this actually gets used in practice. Some things have dictionaryutils pinned like gdcdatamodel which is installing old versions of it. But when dumping the schema it should be safe and desirable to always use the latest

Reply:
Oh. So this is more than just unit tests, it gets run from elsewhere. but it looks like this is getting the version of these utils, not the dictionary itself... is that intended?

Reply:
I could also just be confused about the process

Reply:
I was confused for future reference. This entire utilities repo gets copied into the dictionary and the beginning of this script cd's back into the dictionary